docs/pages/guides/notifications-services.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Notifications
```

Comment:
```suggestion
title: Using Push Notifications
```

Let's change this title to follow the convention in the "using" integration section: 

![CleanShot 2025-02-11 at 00 56 32@2x](https://github.com/user-attachments/assets/9be588d0-7d8a-4bd8-93fc-2e0a9dfc4dc2)